<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoveFile</class>
 <widget class="QWidget" name="MoveFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Move file</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background: #242A38;
	border: 2px solid white;
	border-radius: 2px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background: #4E596F;
    border-width: 1px;
    border-radius: 13px;
    border-color: #4E596F;
    font: 12pt;
    color: white;
    padding: 10px;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 15px;
	margin-bottom: 10px;
	min-height: 20px;
}

QPushButton:hover {
background: qradialgradient(cx:0.3, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #4E596F, stop:1 white);
}

QPushButton:pressed {
border-style: inset;
background: qradialgradient(cx:0.7, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #4E596F, stop:1 black);
}

QPushButton:focus {
   border: none;
   outline: none;
}</string>
       </property>
       <property name="text">
        <string>CANCEL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_confirm">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: #f76163;
    border-width: 1px;
    border-radius: 13px;
    border-color: #f76163;
    font: 12pt;
    color: white;
    padding: 10px;
	margin-left: 0px;
	margin-right: 0px;
	margin-top: 15px;
	margin-bottom: 10px;
	min-height: 20px;
}

QPushButton:hover {
background: qradialgradient(cx:0.3, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 white);
}

QPushButton:pressed {
background: qradialgradient(cx:0.7, cy:0.5, radius: 5.5,
              fx:0.5, fy:0.5, stop:0 #f65964, stop:1 black);
}

QPushButton:focus {
   border: none;
   outline: none;
}</string>
       </property>
       <property name="text">
        <string>CONFIRM</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox_dst">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
                                                    border: 1px solid #333333;
                                                    border-radius: 15px;
                                                    background: #4e596f;
                                                    padding: 1px 0px 1px 40px;
                                                    min-width: 6em;
                                                    font: 14px;
                                                    color: #ffffff;
                                                    }

                                                    QComboBox::drop-down {
                                                    subcontrol-origin: padding;
                                                    subcontrol-position: top right;
                                                    width: 47px;

                                                    border-top-right-radius: 3px;
                                                    border-bottom-right-radius: 3px;

                                                    }

                                                    QComboBox:focus {
                                                    outline: none;
                                                    }

                                                    QComboBox::down-arrow {
                                                    image: url(:/res/dropdown_arrow.png)
                                                    }

                                                    QComboBox QAbstractItemView{
                                                    background-color: #4e596f;
                                                    color: white;

                                                    selection-background-color: #f65964;
                                                    selection-color: white;
                                                    }</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>15</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="text">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid none;
color: white;
font-size: 25px;
qproperty-alignment: 'AlignHCenter|AlignVCenter';</string>
     </property>
     <property name="text">
      <string>Destination folder:</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
